boards/arm/nrf52dk_nrf52832/nrf52dk_nrf52832.yaml

```diff
@@ -20,3 +20,4 @@ supported:
   - pwm
   - spi
   - watchdog
+  - netif
```

Comment:
Do we really want `netif` on this board, it's BLE only?

Reply:
I'd opt for removing the capability here.

Reply:
Done

Reply:
Done